ci/pipelines/deploy-to-test.yml

```diff
@@ -1422,34 +1429,44 @@ jobs:
           AWS_ACCESS_KEY_ID: ((.:role.AWS_ACCESS_KEY_ID))
           AWS_SECRET_ACCESS_KEY: ((.:role.AWS_SECRET_ACCESS_KEY))
           AWS_SESSION_TOKEN: ((.:role.AWS_SESSION_TOKEN))
-      - put: adminusers-latest-ecr-registry-test
-        params:
-          image: adminusers-candidate-ecr-registry-test/image.tar
-#    on_failure:
-#      put: slack-notification
-#      attempts: 10
-#      params:
-#        channel: '#govuk-pay-announce'
-#        silent: true
-#        text: ':red-circle: adminusers candidate image ((.:candidate_image_tag)) failed post-merge end to end tests - <https://pay-cd.deploy.payments.service.gov.uk/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|Concourse build #$BUILD_NAME>'
-#        icon_emoji: ":concourse:"
-#        username: pay-concourse
-#    on_success:
-#      put: slack-notification
-#      attempts: 10
-#      params:
-#        channel: '#govuk-pay-activity'
-#        silent: true
-#        text: ':green-circle: adminusers candidate image ((.:candidate_image_tag)) passed post-merge end to end tests - <https://pay-cd.deploy.payments.service.gov.uk/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|Concourse build #$BUILD_NAME>'
-#        icon_emoji: ":concourse:"
-#        username: pay-concourse
+      - in_parallel:
+        - do:
+          - put: adminusers-ecr-registry-test
+            params:
+              image: adminusers-candidate/image.tar
+              additional_tags: parse-candidate-tag/release-tag
+          - put: adminusers-latest
```

Comment:
Hang on - are we pushing to Dockerhub twice (`latest` here, and `latest-master` below)?

Reply:
No, this is ECR, we are pushing to ecr twice though (it will require a big change to be able to do it in 1 push) but the second push is v quick since it's an already exists, and the two together will take less time than the single push to dockerhub so the flow is

* In parallel
    * in series
        * push X-release
        * push latest
    * push to dockerhub
    
So I still expect the push to ecr tasks to finish prior to the parallel dockerhub

Reply:
Gah... it turns out the line 291 that says `icon: docker` confused me 🤦‍♀️  (that'll serve me right to try and multi-task reviewing big yaml files)

Reply:
Yeah I wish we had an iconset for aws services, would be nice to have an ECR icon :)